cmd/fossa/cmd/test/test.go

```diff
@@ -18,8 +18,20 @@ import (
 	"github.com/fossas/fossa-cli/config"
 )
 
+const defaultTestTemplate = `Test Failed. {{.Count}} {{if gt .Count 1 -}} issues {{- else -}} issue {{- end}} found:
+{{- range $type, $issues := .NormalizedByType}}
+========================================================================
+{{$type}}
+========================================================================
+{{range $i, $issue := $issues}}
+{{$issue.Name}}	{{$issue.Revision}}	{{$issue.Rule.License}}
+{{- end}}
+{{end}}
+`
+
 var Timeout = "timeout"
```

Comment:
can we make all of these constants in a group:
```
const (
	Timeout        = "timeout"
	SuppressIssues = "suppress-issues"
	JSON           = "json"
)
```
Also I don't think any of these need to be exported.

Reply:
Yea you're right, these don't have to be exported. I believe the reason this was done is because the unexported names for a lot of these flags are packages or reserved, such as `json` or `type`. I made these constants, but I'm comfortable leaving them exported for clarity as they are littered around the codebase and want to maintain consistency. Good point though.